cpp/jvm-host/src/host_jni.cpp

```diff
@@ -434,6 +434,15 @@ JNIEXPORT jint JNICALL Java_com_r3_conclave_host_internal_Native_getQuoteDCAP(JN
     return (int)eval_result;
 }
 
+jbyteArray jbyteArrayFromCharPtr(JNIEnv *jniEnv, unsigned size, char* data) {
+    unsigned sizeActual = size - 1; // Drop the \0 terminator
+    jbyteArray array = jniEnv->NewByteArray(sizeActual);
```

Comment:
I always get confused with jni arrays, but would just a  `SetByteArrayRegion` work instead of the code below (just reducing to one call instead of three)?

Reply:
Will give this a try

Reply:
Yup, works fine. Changed!